src/main/java/org/broadinstitute/hellbender/utils/variant/VariantContextVariantAdapter.java

```diff
@@ -24,9 +35,48 @@
     public boolean isSnp() { return variantContext.isSNP(); }
     @Override
     public boolean isIndel() { return variantContext.isIndel(); }
+
+    @Override
+    public UUID getUUID() {
+        return uuid;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        VariantContextVariantAdapter that = (VariantContextVariantAdapter) o;
+
+        // VariantContext doesn't define equality, so we have to.
+        if (!getContig().equals(that.getContig())) {
+            return false;
+        }
+        if (getStart() != that.getStart()) {
+            return false;
+        }
+        if (getEnd() != that.getEnd()) {
+            return false;
+        }
+        if (isSnp() != that.isSnp()) {
+            return false;
+        }
+        if (isIndel() != that.isIndel()) {
+            return false;
+        }
+        return uuid.equals(that.uuid);
+    }
```

Comment:
Not yet -- we will gradually expand the `Variant` interface as needed to expose the other fields.